src/conditional-flow/multiple-conditions.js

```diff
@@ -2,18 +2,16 @@
 // than or equal to lower AND less than or equal to upper.
 // Implement this with a single condition.
 function isInRange (num, lower, upper) {
-
   // TODO: write code in this function body to pass the tests
-
+  return num >= lower && num <= upper ? true : false;
 }
 
 // This function should return true if the passed string is equal
 // to "Hello" or "Goodbye". Implement this with a single
 // if statement.
 function isHelloOrGoodbye (val1) {
-
   // TODO: write code in this function body to pass the tests
-
+  return val1 == 'Hello' || val1 == 'Goodbye' ? true : false;
```

Comment:
Same as above - `return val1 == 'Hello' || var1 == 'Goodbye'` will create a boolean value. Therefore we can just return that value.
```js
return val1 == 'Hello' || var1 == 'Goodbye'
```